index.d.ts

```diff
@@ -120,7 +120,7 @@ export interface CancelTokenSource {
 }
 
 export interface AxiosInterceptorManager<V> {
-  use(onFulfilled?: (value: V) => V | Promise<V>, onRejected?: (error: any) => any): number;
+  use(onFulfilled?: (value: V) => V | Promise<V>, onRejected?: (error: V) => any): number;
```

Comment:
A successful callback should return V（AxiosResponse）, but when there is an error, maybe it should return `AxiosError` insteadof `V`?

Reply:
Are you sure about `AxiosError`?